pkg/minikube/registry/drvs/qemu2/qemu2.go

```diff
@@ -110,8 +110,15 @@ func qemuVersion() (semver.Version, error) {
 	if err != nil {
 		return semver.Version{}, err
 	}
-	v := strings.Split(strings.TrimPrefix(string(rr), "QEMU emulator version "), "\n")[0]
-	return semver.Make(v)
+	versionPattern :=`(\d+\.\d+\.\d+)`
+	re := regexp.MustCompile(versionPattern)
+	matches := re.FindStringSubmatch(string(rr))
+	if len(matches) > 1 {
+		v := matches[1]
+		return semver.Make(v)
+	} else {
+		panic(fmt.Sprintf("Failed to detect semantic version of %s -version output.", qemuSystem))
```

Comment:
why panic ? this func seems to be returning error

Reply:
I understood the situation to be fatal.
I will try to modify the PR to return an error.

Reply:
it would make sense to get representative version strings from the brew version of qemu (I do not use brew) do you have that available?